Added popup box replacement script and branch

jmc317 opened the pull request.

e16johka committed:
Added popup box replacement script  and branch

Kasleets commented on wwwroot/css/site.css:
You meant to input this link as a comment?

You've implemented the script properly inside the site.js already as:
' <script src="https://cdn.jsdelivr.net/npm/sweetalert2@11.9.0/dist/sweetalert2.all.min.js"></script> '

I can't find a usage suggestion inside the sweetalert documentation, is the link intentional?

jmc317 replied:
I linked it like that so that it can be run without having any files in the repository. I asked Jonathan about it and he said it would work ;) The same thing Gill did with bootstrap if I remember correctly

jmc317 replied:
https://www.jsdelivr.com/package/npm/sweetalert2
This is the link I followed

Kasleets reviewed (changes requested):
Only a doublecheck on the link, and then it's fine

e16johka committed:
Fixed the error when launching the app by removing a line from program.cs (Line 19)

Kasleets committed:
Update Program.cs

Don't remove the code without a trace, comment it out, if it is 100% redundant we will remove it during the Code Review
